apps/api/src/app/integrations/usecases/get-In-app-activated/get-In-app-activated.usecase.ts

```diff
@@ -10,7 +10,7 @@ export class GetInAppActivated {
     const inAppSubscriberCount = await this.subscriberRepository.count({
       _organizationId: command.organizationId,
       _environmentId: command.environmentId,
-      isOnline: { $exists: true },
+      isOnline: true,
```

Comment:
Is not this what we really like to check and not only that the isOnline field exists on the subscriber?